### PR TITLE
#2610 - NOA Content update

### PR DESCRIPTION
**Pull request by dheepak-aot**
## Notice of Assessment Content Update

- [x] Update the `This Includes` content based on PT and FT.

Full Time

image

Part Time

image

- [x] Update Miscellaneous costs in PT  by mapping the right key from assessment data.

image

- [x] Added Mandatory Fees to PT to Assessed Costs table.

image

- [x] Create a new variable in camunda for Total Academic Expenses `calculatedDataTotalAcademicExpenses` and map to assessment data to display in NOA

image

image

- [x] Update Total Assessed Costs in PT  by mapping the right key from assessment data.

image

- [x] E2E test for Total Academic Expenses

### CODE REVIEW

File: sources/packages/backend/workflow/test/2023-2024/parttime-assessment/costs/parttime-assessment-2023-2024-costs-total-academic-expenses.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import { OfferingDeliveryOptions } from "@sims/test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-total-academic-expenses.`, () => {
+  it(
+    "Should determine calculatedDataTotalAcademicExpenses when offering weeks is 30, " +
```
Comment (sh16011993) [inline sources/packages/backend/workflow/test/2023-2024/parttime-assessment/costs/parttime-assessment-2023-2024-costs-total-academic-expenses.e2e-spec.ts:10]: Maybe we can use different test data / flows in different program years to have more diversity of the tests.

File: sources/packages/forms/src/form-definitions/noticeofassessment.json [hunk 2]
```diff
@@ -9935,7 +9964,7 @@
                   "tags": [],
                   "properties": {},
                   "conditional": {
-                    "show": true,
+                    "show": "true",
```
Comment (sh16011993) [inline sources/packages/forms/src/form-definitions/noticeofassessment.json:9967]: Is there a specific reason for the double-quotes here?

**Review by sh16011993 — APPROVED** [review]
Nice Work @dheepak-aot 👍 . Thanks for the clarifications on the call.

File: sources/packages/backend/workflow/test/2021-2022/parttime-assessment/costs/parttime-assessment-2021-2022-costs-total-academic-expenses.e2e-spec.ts
Comment (andrewsignori-aot) [inline sources/packages/backend/workflow/test/2021-2022/parttime-assessment/costs/parttime-assessment-2021-2022-costs-total-academic-expenses.e2e-spec.ts:1]: Can we have an agreement to stop adding the program year in the file names?
We need to go back and rename the existing ones but I do not see a reason to keep adding them right now.

**Review by lewischen-aot — APPROVED** [review]
LGTM @dheepak-aot Nice work 👍

**Review by andrewsignori-aot — APPROVED** [review]
Great work, looks good 👍

**Pull request merged**